static/css/main.css

```diff
@@ -19,3 +19,8 @@ html {
 .card-deck .card {
   min-width: 220px;
 }
+
+.logo {
+  height: 1.5em;
```

Comment:
Mixing ems and pixels might not be the most approachable way to lay this out. Does this have a predictable and static pixel count?

Reply:
Hm, this is an area I'm super-weak on, so I don't know.